lib/linguist/languages.yml

```diff
@@ -449,6 +449,14 @@ Cap'n Proto:
   - .capnp
   ace_mode: text
 
+CartoCSS:
+  aliases:
+  - Carto
+  extensions:
+  - .mss
+  ace_mode: text
```

Comment:
OK thanks for checking @floscher. Let's wait until TextMate-grammar is available - do you know of any in development?